src/django/api/views/v1/opensearch_query_builder/opensearch_query_director.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, builder):
             V1_PARAMETERS_LIST.ADDRESS: 'match',
             V1_PARAMETERS_LIST.NAME: 'match',
             V1_PARAMETERS_LIST.OS_ID: 'terms',
-            V1_PARAMETERS_LIST.NAME_LOCAL: 'match',
+            V1_PARAMETERS_LIST.LOCAL_NAME: 'terms',
```

Comment:
Was this change purposeful or accidental? I had previously changed this value to `match`.

Reply:
It was the merge mistake. Changed back. Thank you.